add missing channel names (fixes a segfault when parsing invalid channel maps)

From: Lennart Poettering <lennart@fefdeb5f-60dc-0310-8127-8f9354f1896f>

git-svn-id: svn://svn.0pointer.net/pulseaudio/trunk@1378 fefdeb5f-60dc-0310-8127-8f9354f1896f
---

 src/pulse/channelmap.c |   16 ++++++++++++++++
 1 files changed, 16 insertions(+), 0 deletions(-)

diff --git a/src/pulse/channelmap.c b/src/pulse/channelmap.c
index a4f1337..69b0908 100644
--- a/src/pulse/channelmap.c
+++ b/src/pulse/channelmap.c
@@ -68,6 +68,22 @@ const char *const table[] = {
     [PA_CHANNEL_POSITION_AUX13] = "aux13",
     [PA_CHANNEL_POSITION_AUX14] = "aux14",
     [PA_CHANNEL_POSITION_AUX15] = "aux15",
+    [PA_CHANNEL_POSITION_AUX16] = "aux16",
+    [PA_CHANNEL_POSITION_AUX17] = "aux17",
+    [PA_CHANNEL_POSITION_AUX18] = "aux18",
+    [PA_CHANNEL_POSITION_AUX19] = "aux19",
+    [PA_CHANNEL_POSITION_AUX20] = "aux20",
+    [PA_CHANNEL_POSITION_AUX21] = "aux21",
+    [PA_CHANNEL_POSITION_AUX22] = "aux22",
+    [PA_CHANNEL_POSITION_AUX23] = "aux23",
+    [PA_CHANNEL_POSITION_AUX24] = "aux24",
+    [PA_CHANNEL_POSITION_AUX25] = "aux25",
+    [PA_CHANNEL_POSITION_AUX26] = "aux26",
+    [PA_CHANNEL_POSITION_AUX27] = "aux27",
+    [PA_CHANNEL_POSITION_AUX28] = "aux28",
+    [PA_CHANNEL_POSITION_AUX29] = "aux29",
+    [PA_CHANNEL_POSITION_AUX30] = "aux30",
+    [PA_CHANNEL_POSITION_AUX31] = "aux31",
 
     [PA_CHANNEL_POSITION_TOP_CENTER] = "top-center",
     
